bot/bot.py

```diff
@@ -57,30 +72,110 @@ async def register(inter: ApplicationCommandInteraction, nickname: str, password
+
+@bot.slash_command(name="logout", description="Logout from current account")
+async def logout(inter: ApplicationCommandInteraction) -> None:
+        
         await inter.response.defer(ephemeral=True)
-        if not await database.is_password_and_nickname_valid(nickname, password, str(inter.author.id)):
-            embed = Embed(title="Confession",
-                        description=f"Sorry, the nickname: `{nickname}`, does not exist in the database.",
-                        timestamp=datetime.now(timezone))
-            await inter.edit_original_response(embed=embed)
-            return 
-    
-        await database.unregister_nickname(nickname, password, str(inter.author.id))
+        valid = await database.logout(str(inter.author.id))
+        if valid is False:
+            embed = Embed(title="Error", description="You are not logged in.", color=0xFF0000)
+            await inter.edit_original_message(embed=embed)
+            return
+        
         embed = Embed(title="Confession",
-                    description=f"RIP 🥀 {nickname}\n\nFly high butterfly 🥹\n\n🕊️ Died peacefully in {datetime.now(timezone).strftime('%b %d, %Y')}")
+                    description="Successfully logged out",
+                    timestamp=datetime.now(timezone))
     
-        await inter.channel.send(embed=embed)
+        await inter.edit_original_response(embed=embed)
+        return
+
+
+@bot.slash_command()
+async def user(inter):
+    pass
+
+@user.sub_command(name="delete", description="Delete current nickname")
+async def delete(inter: ApplicationCommandInteraction, password: str) -> None:
+
+    await inter.response.defer(ephemeral=True)
+    nickname = await database.get_nickname(str(inter.author.id))
+    if nickname is None:
+        embed = Embed(title="Error", description="You don't have a nickname yet. Please register one using `/register` or if you have one already, login via '/login'.", color=0xFF0000)
+        await inter.edit_original_message(embed=embed)
         return
     
-@bot.slash_command(name="changepassword", description="Change password for a nickname")
-async def change_password(inter: ApplicationCommandInteraction, nickname: str, old_password: str, new_password: str) -> None:
+    if not await database.is_password_and_nickname_valid(nickname, password, str(inter.author.id)):
+        embed = Embed(title="Confession",
+                    description=f"Sorry, the nickname: `{nickname}`, does not exist in the database.",
+                    timestamp=datetime.now(timezone))
+        await inter.edit_original_response(embed=embed)
+        return 
+
+    await database.delete_nickname(nickname, password, str(inter.author.id))
+    embed = Embed(title="Confession",
+                description=f"RIP 🥀 {nickname}\n\nFly high butterfly 🥹\n\n🕊️ Died peacefully in {datetime.now(timezone).strftime('%b %d, %Y')}")
+
+    await inter.channel.send(embed=embed)
+    return
+
+@user.sub_command_group()
+async def change(inter):
```

Comment:
Walang types

Reply:
wala ng param ata dapat

Reply:
https://docs.disnake.dev/en/stable/ext/commands/slash_commands.html#subcommands-and-groups

Reply:
-> None ?

Reply:
May type 'yong inter